### ISSUE TITLE
Throw better error when workspace member doesn't exist

### ISSUE THREAD

**Issue by ChewingGlass** [project-serum/anchor#1492]
If you have a path in Anchor.toml workspace.members that doesn't exist, it throws:

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', cli/src/config.rs:191:22
```

A better error should be thrown here

**Comment by anantasty** [comment]
@armaniferrante I would like to take this as my first issue.

**Comment by paul-schaaf** [comment]
closed by #1524
